loaders/httpfs/loader.go

```diff
@@ -0,0 +1,30 @@
+package httpfs
+
+import (
+	"io"
+	"net/http"
+)
+
+type httpFileSystemLoader struct {
+	http.FileSystem
+}
+
+func NewLoader(fs http.FileSystem) httpFileSystemLoader {
```

Comment:
@jhsx I moved the osfs pkg into the Jet core, as you suggested. Not sure how to fix the import issue, though. We could move the tests into a sub-directory - or just leave the the `jet.Loader` interface as return value for the time being.

What do you think?